ws-nextjs-app/cypress/e2e/livePage/index.cy.ts

```diff
@@ -0,0 +1,28 @@
+describe(
+  'Live Page Spec',
+  {
+    env: {
+      optionalOvveride: 'foo',
+    },
+  },
+  // @ts-expect-error Conflicts with Jest global types describe, it etc.
+  () => {
+    // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
+    let livePageData;
+    before(() => {
+      cy.getPageData({
+        service: 'pidgin',
+        pageType: 'live',
+        id: 'c7p765ynk9qt',
+      }).then(({ body }) => {
+        livePageData = body;
+      });
+    });
+
+    if (Cypress.env('APP_ENV') === 'test') {
```

Comment:
Does this only work on test because we don't have fixture data in local for live pages, like we do for the other page types? Wondering if that might be worth considering adding, given that we would like to be able to run these E2Es on PRs to confirm that everything is OK before merging. Worth a shot?